generic/Storage.c

```diff
@@ -47,7 +68,7 @@ static int torch_Storage_(new)(lua_State *L)
       storage = THStorage_(newWithDataAndAllocator)(ptr, size, allocator, NULL);
     else
       storage = THStorage_(newWithData)(ptr, size);
-    storage->flag = 0;
+    storage->flag = TH_STORAGE_REFCOUNTED;
```

Comment:
@clementfarabet What did you mean by this? Its not changed like this in cutorch.